### ISSUE TITLE
Add guidance about keeping panel heading brief

### ISSUE THREAD

**Issue by hannalaakso** [alphagov/govuk-design-system#1901]
## What

We fixed the panel heading text potentially overflowing its container when user zooms in on a mobile device at a high magnification in https://github.com/alphagov/govuk-frontend/pull/2347.

We could consider adding something to the [panel guidance](https://design-system.service.gov.uk/components/panel/) about how the **heading** text should be kept to the point and (if it isn't too weird to specify this in the guidance) using shorter words as these are less likely to wrap, and how the description text below should be used to give more detailed information and context about the transaction.

## Why

This will help to mitigate the heading text wrapping when user zooms in on a mobile device at a high magnification. 

Additionally, the heading text is only meant to give a high level overview of what has happened so it makes sense to keep the wording snappy.

## Who needs to know about this

Content writer, possibly a developer

## Done when

- [ ] We have added some guidance to the Design System about the need to keep the panel heading text brief, and possibly something about how the description text below should be used to give more detailed information and context
